src/content/blog/astro-530.mdx

```diff
@@ -0,0 +1,93 @@
+
+The exact amount of the improvement will depend on the individual sites. Complex pages with many components will see the biggest improvements, while pages that use slower external APIs at request time may be unchanged. In our testing, SSR responses are generally 1.5-2x faster unless the page is querying external APIs. Some examples:
+
+- For the 5300-page [Astro docs site](https://docs.astro.build), it **cut static build times by 10-15%.** 
+- In the Platformic SSR benchmark, which generates a page with over 3000 components, requests are **4x faster**.
+
+You don't need to do anything to benefit from this improvement: just sit back and enjoy the faster rendering times.
+
+## Automatic session storage setup
+
+Astro 5.1 introduced the experimental [session storage API](https://docs.astro.build/en/reference/experimental-flags/sessions/), which allows developers to easily manage user sessions on the server. The feature supports dozens of different backend storage drivers, powered by [unstorage](https://unstorage.unjs.io). Previously you would need to configure the session storage driver in your `astro.config.mjs`, but now Astro will automatically set up the session storage for you when using a supported adapter. 
+
+Currently, these supported adapters include `@astrojs/node`, which uses the filesystem for storage, and `@astrojs/netlify`, which uses Netlify Blobs. We hope to add support for more adapters in the near future, where the hosting platforms include a storage API. In all cases, you can still manually configure your own session storage driver if you prefer.
+
+To enable session storage, set the `experimental.session` flag in your `astro.config.mjs`:
+
+```js title="astro.config.mjs" ins={6}
+  {
+    adapter: node({
+      mode: "standalone",
+    }),
+    experimental: {
+      session: true,
+    },
+  }
+```
+
+If you are already using the session feature, note that the syntax for the config has changed. The `experimental.session` option is now a boolean flag, and the actual session config has moved to a top-level `session` key. This allows you to enable the feature without needing to configure the session storage driver when on a platform that configures it for you.
+
+```js title="astro.config.mjs" ins={9,11-13} del={6-8}
+  {
+    adapter: node({
+      mode: "standalone",
+    }),
+    experimental: {
+        session: {
+            driver: "upstash",
+        },
+        session: true,
+    },
+    session: {
+        driver: "upstash",
+    },
+  }
+```
+
+## Bug fixes
```

Comment:
Up to you if you want a blurb about the Netlify adapter updates here. Since this change adds a feature that the Vercel adapter already had, it might be nice to mention the improvement here!